index.bs

```diff
@@ -8488,33 +8341,6 @@ Compile the graph up to the output operand.
 </pre>
 </div>
 
-<div class="example">
-<details open>
-  <summary>
-    The following code executes the compiled graph.
-  </summary>
-  <pre highlight="js">
-    // Setup the input buffers with value 1.
-    const inputBuffer1 = new Float32Array(TENSOR_SIZE).fill(1);
-    const inputBuffer2 = new Float32Array(TENSOR_SIZE).fill(1);
-    const outputBuffer = new Float32Array(TENSOR_SIZE);
-
-    // Execute the compiled graph with the specified inputs.
-    const inputs = {
-      'input1': inputBuffer1,
-      'input2': inputBuffer2,
-    };
-    const outputs = {
-      'output': outputBuffer
-    };
-    const result = await context.compute(graph, inputs, outputs);
-
-    console.log('Output value: ' + result.outputs.output);
-    // Output value: 2.25,2.25,2.25,2.25,2.25,2.25,2.25,2.25
-  </pre>
-</details>
-</div>
```

Comment:
I considered re-creating this example with `dispatch()`... but it would end up looking a whole lot like https://webmachinelearning.github.io/webnn/#api-mlcontext-dispatch-examples. Is that still useful? Open to suggestions

Reply:
It seems to be duplicate. Should we also remove other examples of getting context and building graph in https://webmachinelearning.github.io/webnn/#examples? They also seem to be duplicated by dispatch example.

Reply:
Hmm. Yeah examples 6 and 8 (which I assume are the ones you're referring to) don't seem very useful. Example 7 ("The following code implements the graph....") seems pretty useful, though, especially since it gives a visual of how the operands relate to each other

Tentatively removed examples 6 and 8 in the latest patch b94b334

Reply:
> Example 7 ("The following code implements the graph....") seems pretty useful, though, especially since it gives a visual of how the operands relate to each other

Agreed.

Reply:
But keeping example 7 alone would make it incomplete, could we add `const context = await navigator.ml.createContext();` before `const builder = new MLGraphBuilder(context);`?

Reply:
Done in efdc1e8

Reply:
I was going to point this out too. Examples 6, 7, 8, 9 are all part of a series, and so deleting just parts of it make it incomplete, but I suppose so long as https://webmachinelearning.github.io/webnn/#api-mlcontext-dispatch-examples exists, then we at least have a whole example in the spec still.